app/domain/authentication/authn_jwt/validate_request_body.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+
+# This class validates that request body contains JWT encoded token
+module Authentication
+  module AuthnJwt
+
+    ValidateRequestBody ||= CommandClass.new(
+      dependencies: {
+      },
+      inputs: %i[body_string]
+    ) do
+
+      def call
+        validate_body
+      end
+
+      private
+
+      def validate_body
```

Comment:
i would call this `validate_body_is_jwt` to be more explicit. if we will have more checks in the future they can be in different methods